ci/ansible/roles/pulp-crane/tasks/main.yml

```diff
@@ -30,9 +43,33 @@
     group: root
     mode: 0644
```

Comment:
@Ichimonji10 
Ok I changed the task name to disambiguate. Does that make more sense now? EL6 needs an additional config file.